tests/Fixer/PhpTag/NoClosingTagFixerTest.php

```diff
@@ -174,14 +174,5 @@ public static function provideWithShortOpenTagCases(): iterable
         yield [
             '<? /**/', '<? /**/?>',
         ];
-
-        yield [
-            '<?= "somestring"; ?> <?= "anotherstring"; ?>',
-        ];
-
-        yield [
-            '<?= 1;',
-            '<?= 1; ?>',
-        ];
```

Comment:
These test cases look like proper ones 🤔.

Reply:
related with https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7687#discussion_r1445117767 unification, see all other fixers code for more global overview